### PR TITLE
Food deletion

### LINKED ISSUE #5: User can delete a food
User can delete food by sending food id to `DELETE /api/v1/foods/:id` which deletes the food from the database.

### Checks
- [x] For success, returns status code 204

Sad path
- [x] For error, returns status code 404 if food cannot be found

### PR DESCRIPTION
**Pull request by Mycobee**
## What functionality does this accomplish?
closes #5

**Description:**

User can delete food by sending food id to `DELETE /api/v1/foods/:id`

## What did you struggle on to complete?

No blockers for this endpoint

### Test Coverage Percentage: 100%
- [ ] No Tests have been changed
- [x] Some Tests have been changed
- [ ] All of the Tests have been changed(Please describe what in the world happened):

## Checklist:
- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [x] I have fully tested my code
- [ ] I have partially tested my code (please explain why):

## Helpful Resources:
* Resource link AND small description of info gathered/learned
[Turing Server Side JS Lesson](https://backend.turing.io/module4/lessons/server_side_javascript)
[Turing Jest Testing](https://backend.turing.io/module4/lessons/jest_testing)

### CODE REVIEW

File: routes/api/v1/foods.js [hunk 1]
```diff
@@ -16,4 +16,24 @@ router.get("/", function (req, res) {
...
 });
 
+router.delete('/:id', function(req, res, next) {
+  Food.destroy({
+    where: {
+      id: req.params.id
+    }
+  })
+	.then(food => {
+     if (food !== 1)
```
Comment (lpile) [inline routes/api/v1/foods.js]: Hey Rob, I don't think the if statement is working as intended or I don't understand what it's doing. I played around with your branch and I cannot get inside this block. It seems to just to the catch every time.

File: routes/api/v1/foods.js [hunk 2]
```diff
@@ -16,4 +16,24 @@ router.get("/", function (req, res) {
...
+	{throw new Error("Not Found")}
+	})
+  .then(food => {
+    res.setHeader('Content-Type', 'application/json');
+    res.status(204).send(JSON.stringify(food));
```
Comment (lpile) [inline routes/api/v1/foods.js]: With a status code 204, you don't need to send anything back.

File: routes/api/v1/foods.js (hunk 1 above)
Comment (Mycobee) [reply routes/api/v1/foods.js]: You are right that it isn't clear.  Apparently food returns 1 if it has been deleted, so this conditional throws an error when something hasn't been deleted.  I will revise it to be cleaner.

File: routes/api/v1/foods.js (hunk 2 above)
Comment (Mycobee) [reply routes/api/v1/foods.js]: Cool.  I will remove it

**Pull request merged**